web/testing/promise-api/promisehdr.js

```diff
@@ -49,9 +49,13 @@
   function addKeyboardsForLanguage(languages) {
     kmw.addKeyboardsForLanguage(languages.split(',').map(item => item.trim())).then(result => {
       console.log('Adding ' + languages + ': ', result);
```

Comment:
This - the `then` clause - _will_ receive any `ErrorStubs` - you may wish to filter them out by checking on the conditional existence of the `error` field on each entry.

Reply:
I tried that in https://github.com/keymanapp/keyman/pull/5340/commits/4efad838534e73af89665e729b350f8e8fa7fc33

But I find it easier in testing to see all the returned stubs in `result`.